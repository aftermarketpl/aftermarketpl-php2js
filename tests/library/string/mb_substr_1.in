$a = "ąĆęŁ";
return mb_substr($a, 2);
