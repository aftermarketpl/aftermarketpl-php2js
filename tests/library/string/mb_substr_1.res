"\u0119\u0141"