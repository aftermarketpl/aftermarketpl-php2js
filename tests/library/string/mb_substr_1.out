a = "ąĆęŁ";
return (a).substr(2);
